up-linux-streamer/src/main.rs

```diff
@@ -55,16 +53,19 @@ async fn main() -> Result<(), UStatus> {
         usubscription,
     );
 
-    let mut zenoh_config = ZenohConfig::default();
-
-    // Specify the address to listen on using IPv4
-    let ipv4_endpoint = ZenohEndpoint::from_str(config.zenoh_transport_config.endpoint.as_str());
-
-    // Add the IPv4 endpoint to the Zenoh configuration
-    zenoh_config
-        .listen
-        .endpoints
-        .push(ipv4_endpoint.expect("FAIL"));
+    let zenoh_config = match ZenohConfig::from_file(config.zenoh_transport_config.config_file) {
+        Ok(config) => {
+            trace!("Able to read zenoh config from file");
+            config
+        }
+        Err(error) => {
+            warn!(
```

Comment:
IMHO we should panic if it's not found. We should fail very obviously. I can imagine a case where someone has difficulty figuring out why the streamer is not working due to this.

Reply:
makes sense, changed